### ISSUE TITLE
Timing Equivalence

### ISSUE THREAD

**Issue by obruendl** [risclite/R8051#4]
Does the core guarantee timing equivalence (i.e. that every instruction takes exactly the same amount of clock cycles as in a original 8051 processor)?

**Comment by risclite** [comment]
No, it is based on byte flow. In another word, how many bytes one instruction has, how many clock cycles it has.
